Doc/library/sys.rst

```diff
@@ -449,10 +449,9 @@ always available.
 
 .. function:: exit([arg])
 
-   Exit from Python.  This is implemented by raising the :exc:`SystemExit`
-   exception, so cleanup actions specified by finally clauses of :keyword:`try`
-   statements are honored, and it is possible to intercept the exit attempt at
-   an outer level.
+   Raise a :exc:`SystemExit` exception, signaling an intention to exit the interpreter.
+   Cleanup actions specified by finally clauses of :keyword:`try` statements are honored, 
+   and it is possible to intercept the exit attempt at an outer level. 
```

Comment:
I'd move the second sentence ("Cleanup...") down - just after the sentence that ends in line 471.

Reply:
Yup. The document is updated. please review.